home/storage/LocalStorage.js

```diff
@@ -49,7 +49,14 @@ async function updateSettingsAsync(updatedSettings) {
 }
 
 async function getSessionAsync() {
-  let results = await AsyncStorage.getItem(Keys.Session);
+  let results = await ExponentKernel.getSessionAsync();
+  if (!results) {
+    // try querying legacy location
+    results = await AsyncStorage.getItem(Keys.Session);
+    if (results) {
+      await AsyncStorage.removeItem(Keys.Session);
```

Comment:
We need to also write the value to the new session store

Reply:
this isn't necessary in the current form of the code because the only place where `getSessionAsync` is called also calls `saveSessionAsync` immediately afterwards. But this is a good idea, I've added it for future proofing.